### PR TITLE
Feat/nft design changes

### PR DESCRIPTION
**Pull request by rsteubs-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

Screen Shot 2022-06-28 at 7 35 56 AM

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Buy/cta.tsx [hunk 1]
```diff
@@ -174,7 +174,16 @@ const CTA: React.FC<Props> = (props) => {
         ),
         Success: (val) => (
           <div>
-            <div style={{ display: 'flex' }}>
+            <div
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Buy/cta.tsx:177]: Please create an styled component for this

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/cta.tsx [hunk 2]
```diff
@@ -139,7 +140,16 @@ const CTA: React.FC<Props> = ({
         </Text>
       ) : null}
       {disabled ? null : (
-        <div style={{ display: 'flex' }}>
+        <div
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/cta.tsx:143]: Same here, also many styles are being duplicated, you could define an styled component in a `.model` file for being reused

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Buy/cta.tsx [hunk 3]
```diff
@@ -174,7 +174,16 @@ const CTA: React.FC<Props> = (props) => {
         ),
         Success: (val) => (
           <div>
-            <div style={{ display: 'flex' }}>
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Buy/cta.tsx:177]: Try using `<Flex>` component

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/Stats/index.tsx [hunk 4]
```diff
@@ -40,16 +40,16 @@ const Stats: React.FC<Props> = ({ formActions, routerActions, slug, stats, total
           <Text size='16px' weight={500} color='grey600'>
             <FormattedMessage id='copy.owners' defaultMessage='Owners' />
           </Text>
-          <Text size='16px' color='white' weight={600}>
+          <Text size='16px' color='black' weight={600}>
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/Stats/index.tsx:43]: why did change the comment code? also, there should not be committed commented code

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/components/NftAssetItem.tsx [hunk 5]
```diff
@@ -124,17 +123,21 @@ const NftAssetItem: React.FC<Props> = ({ asset }) => {
...
-            image_url={image}
-          />
         </Link>
       </LinkContainer>
       <AssetDetails>
         <Flex flexDirection='column' gap={8}>
-          <Text size='18px' style={{ marginTop: '4px' }} color='white' weight={600}>
+          <Text
+            size='18px'
+            style={{
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/components/NftAssetItem.tsx:132]: Too many inline styles, a good practice for code readability would be to move to a styled component

**Review by julianromero-bc — COMMENTED** [review]
Could you add to the PR a gift or an image for visual changes?

File: packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/cta.tsx [hunk 6]
```diff
@@ -139,7 +142,7 @@ const CTA: React.FC<Props> = ({
         </Text>
       ) : null}
       {disabled ? null : (
-        <div style={{ display: 'flex' }}>
+        <CheckboxWraper termsAccepted={termsAccepted}>
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/MakeOffer/cta.tsx:145]: CheckboxWrapper (two ps)

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/Collection.tsx [hunk 7]
```diff
@@ -80,6 +91,15 @@ const LinksContainer = styled.div`
   }
 `
 
+const OuterCollectionInfo = styled.div`
+  position: sticky;
+  top: calc(56px);
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/Collection.tsx:96]: unnecessary calc

Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/Collection.tsx:96]: should you use a constant here like `FIXED_HEADER_HEIGHT`?

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/Collection.tsx [hunk 8]
```diff
@@ -80,6 +91,15 @@ const LinksContainer = styled.div`
...
+  display: flex;
+  z-index: 21;
+  background: white;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Collection/Collection.tsx:99]: don't hardcode

File: packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Firehose/Firehose.tsx [hunk 9]
```diff
@@ -75,7 +75,7 @@ const NftFirehose: React.FC<Props> = ({ formActions, formValues }) => {
       />
       <div style={{ width: '100%' }}>
         <TraitGridFilters
-          collections={[]}
+          collections={collectionsQuery.data?.collections || []}
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/Nfts/Firehose/Firehose.tsx:78]: ah, thanks :)

**Review by plondon — APPROVED** [review]

**Pull request merged**